src/pages/contact.cljs

```diff
@@ -56,11 +57,12 @@
 
 
 (defn content []
- [(dv-map)
+ (layout.rounded-corners/wrapper
```

Comment:
why don't you put this where `content` is called rather than trying to put it inside each `content`?